cluster-api-gcp/deploy/deploy.go

```diff
@@ -40,14 +39,14 @@ func NewDeployer(provider string, configPath string) *deployer {
 	if configPath == "" {
 		configPath = apiutil.GetDefaultKubeConfigPath()
 	}
-	a, err := cloud.NewMachineActuator(provider, token, nil)
+	a, err := newMachineDeployer(provider, token)
```

Comment:
the var name `a` no longer makes sense. please rename to `md` or something like that